java/lang/security/audit/formatted-sql-string.java

```diff
@@ -117,3 +119,16 @@ public void tableConcat() {
         stmt.execute(String.format("CREATE TABLE %s", tableName));
     }
 }
+
+// This whole operation has nothing to do with SQL
+public class FalsePositiveCase {
+    private ApiClient apiClient; // imagine an ApiClient class that contains a method named execute
+
+    public void test(String parameter) throws ApiException {
+        com.squareup.okhttp.Call call = constructHttpCall(parameter); // Create OKHttp call using parameter from outside
+        apiClient.execute(call);
+        apiClient.execute(call);
+        apiClient.run(call); // proof that 'execute' name is causing the false-positive
```

Comment:
These should have `//ok: formatted-sql-string` annotations

Reply:
ah, good catch 👍